torch/_torch_docs.py

```diff
@@ -5787,3 +5787,36 @@ def parse_kwargs(desc):
     Tensor: A 1-D tensor of size :math:`(\text{{window_length}},)` containing the window
 
 """.format(**factory_common_args))
+
+
+add_docstr(torch.meshgrid,
+           r"""
+meshgrid(seq) -> seq
+
+Take a sequence of :math:`N` tensors, each of which be either scalar or 1-dimensional vector,
+and create :math:`N` grids, where the :math:`i`th grid is defined by expanding the :math:`i`th
```

Comment:
nit: "and create :math:`N` N-dimensional grids" (this is the correct behavior, yes?)

Reply:
Yes, it is.